Extension/src/LanguageServer/configurations.ts

```diff
@@ -681,11 +699,6 @@ export class CppProperties {
                 }
 
                 let fullPathToFile: string = path.join(this.configFolder, "c_cpp_properties.json");
-                let filePath: vscode.Uri = vscode.Uri.file(fullPathToFile).with({ scheme: "untitled" });
-
-                let document: vscode.TextDocument = await vscode.workspace.openTextDocument(filePath);
```

Comment:
I was thinking that this would open and (potentially) focus the file in the editor.  Are we doing anything to maintain that behavior with this new approach?